recipes/pyinterp/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyinterp" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CNES/pangeo-{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51660edb108c7ecb9f47741f0c7b51b4b22236e133d2e1b963cdfab4fcc6ae57
+
+build:
+  number: 0
+  skip: true  # [linux32 or win32 or py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [linux]
+    - cmake
+  host:
+    - eigen
+    - gsl
+    - libboost
+    - numpy >=1.11
```

Comment:
Use just `numpy` here. Otherwise your package will be compatible with only the latest version of numpy

Reply:
ok